src/main.cc

```diff
@@ -1725,7 +1730,16 @@ Parse(TRITONSERVER_ServerOptions** server_options, int argc, char** argv)
         cuda_pools.push_back(ParsePairOption<int, uint64_t>(optarg, ":"));
         break;
       case OPTION_RESPONSE_CACHE_BYTE_SIZE:
-        response_cache_byte_size = (uint64_t)ParseLongLongOption(optarg);
+        std::cerr
+            << "Warning: '--response-cache-byte-size' has been deprecated! "
```

Comment:
Should we return an error here or make some changes to make it backward compatible?

Reply:
Agree, should still set it if user specifies it. Internally set the value in cache config if not provided and use local cache